mis_builder/static/src/components/mis_report_widget.xml

```diff
@@ -47,6 +51,7 @@
                             <th
                                 t-foreach="row.cols"
                                 t-as="col"
+                                t-key="col_index"
```

Comment:
Is this col_index used anywhere?

Reply:
I won't be used anywhere, but in OWL 2.0 in Odoo 16, OWL required t-key tag for t-foreach ([ref](https://github.com/odoo/owl/blob/master/CHANGELOG.md#20-t-foreach-should-always-have-a-corresponding-t-key))